php/5.6/Dockerfile

```diff
@@ -30,22 +30,11 @@ RUN sed -i "s/;date.timezone =/date.timezone = Etc\/UTC/" /etc/php5/cli/php.ini
 
 RUN php5dismod xdebug
 
-# Add a "docker" user
```

Comment:
No. You can remove the `USER docker` steps, but we need to keep the user in the images.